doc/classes/Particles2D.xml

```diff
@@ -16,12 +16,14 @@
 			<return type="Rect2">
 			</return>
 			<description>
+				Returns a [code]Rect2[/code] which contains the location and size of the rectangle where the particles are currently present.
```

Comment:
This sentence is too verbose. The return value indicates `Rect2`, so it doesn't have to be repeated in the description. "Rect2 containing the location and size of the rectangle" is very redundant, and somewhat confusing.

How about:

> Returns a rectangle containing the positions of all existing particles.